PyTDF/backend/Local.py

```diff
@@ -0,0 +1,25 @@
+import ROOT
+
+def local_executor(generator):
+    """
+    Execution of the event-loop
+    in local environment
+
+    """
+    mapper = generator.get_mapper_callable()
+    TChain = ROOT.TChain(generator.root_node.treename)
+
+    for f in generator.root_node.filelist:
+        TChain.Add(f)
+
+    TDF = ROOT.Experimental.TDataFrame(TChain)
+
+    generator.root_node.TChain = TChain
+    generator.root_node.TDF = TDF
+
+    output = mapper(generator.root_node.TDF)
+
+    node_map = generator.action_node_map
+
+    for n in node_map:
+        node_map[n].value = output[n]
```

Comment:
@shravan97 where is the event loop triggered?

Reply:
The event loop is actually triggered inside `Proxy.py` , where the user applies `GetValue ()` to an action node

Reply:
that's too late, it wouldn't work in a distributed execution